example/src/main/java/org/apache/rocketmq/example/rpc/ResponseConsumer.java

```diff
@@ -33,46 +31,50 @@
 import org.apache.rocketmq.remoting.exception.RemotingException;
 
 public class ResponseConsumer {
+
+    public static final String PRODUCER_GROUP = "please_rename_unique_group_name";
+    public static final String CONSUMER_GROUP = "please_rename_unique_group_name";
+    public static final String DEFAULT_NAMESRVADDR = "127.0.0.1:9876";
+    public static final String TOPIC = "RequestTopic";
+
     public static void main(String[] args) throws InterruptedException, MQClientException {
-        String producerGroup = "please_rename_unique_group_name";
-        String consumerGroup = "please_rename_unique_group_name";
-        String topic = "RequestTopic";
 
         // create a producer to send reply message
-        DefaultMQProducer replyProducer = new DefaultMQProducer(producerGroup);
+        DefaultMQProducer replyProducer = new DefaultMQProducer(PRODUCER_GROUP);
+        // You need to set namesrvAddr to the address of the local namesrv
+        replyProducer.setNamesrvAddr(DEFAULT_NAMESRVADDR);
         replyProducer.start();
 
         // create consumer
-        DefaultMQPushConsumer consumer = new DefaultMQPushConsumer(consumerGroup);
+        DefaultMQPushConsumer consumer = new DefaultMQPushConsumer(CONSUMER_GROUP);
+        // You need to set namesrvAddr to the address of the local namesrv
+        consumer.setNamesrvAddr(DEFAULT_NAMESRVADDR);
         consumer.setConsumeFromWhere(ConsumeFromWhere.CONSUME_FROM_LAST_OFFSET);
```

Comment:
May cause the official website quick start to set namesrv through the command line to fail.

![image](https://user-images.githubusercontent.com/14222167/169222247-c94c3a0d-2a74-4a72-9253-afdaf04e313c.png)